app/code/Magento/InventoryLowStockNotification/Test/_files/low_stock_notifications.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
You can expand current `app\code\Magento\InventoryLowQuantityNotificationApi\Test\_files\source_item_configuration.php`